core/platform/search/gae_search_services_test.py

```diff
@@ -263,6 +263,11 @@ def test_put_error_without_transient_result(self):
         self.assertEqual(add_docs_counter.times_called, 1)
         self.assertEqual(e.exception.original_exception, error)
 
+    def test_raise_error_when_document_type_is_invalid(self):
+        with self.assertRaises(ValueError):
+            doc = {'abc': set('xyz')}
```

Comment:
Pull this out of the "with" clause. Have that clause be minimal so that you know where the ValueError is coming from.

Reply:
done